test/Microsoft.AspNetCore.SignalR.Tests/DefaultHubLifetimeManagerTests.cs

```diff
@@ -128,7 +128,37 @@ public class DefaultHubLifetimeManagerTests
         }
 
         [Fact]
-        public async Task InvokeConnectionAsyncOnNonExistentConnectionNoops()
+        public async Task SendAllAsyncSendsToAllConnectionsEvenWhenSomeFailToSend()
```

Comment:
Should probably add a test for each method I changed

Reply:
Yeah, and unfortunately it doesn't really work with a Theory.